tensorboard/dataclass_compat.py

```diff
@@ -37,7 +40,7 @@
 from tensorboard.util import tensor_util
 
 
-def migrate_event(event):
+def migrate_event(event, experimental_filter_graph=False):
```

Comment:
Please add doc string for the new kwarg.

Reply:
Oops, done.